src/makeInfinityAdder.js

```diff
@@ -6,7 +6,29 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  return (a) => {
+    let sum = a;
+
+    if (!a) {
+      return 0;
+    }
```

Comment:
You can remove this function and have only one function getSum inside makeInfinityAdder. Because in getSum  you return the function and your code will work fine without this one function.

Reply:
yes, now I see, thanks!)